docs/sdk/1-workshop/02-overview.md

```diff
@@ -8,42 +8,47 @@ In this tutorial, we'll run through the full story of a claim.
 To do so, three actors will be involved: a <span class="label-role claimer">Claimer</span>, an <span class="label-role attester">Attester</span> and a <span class="label-role verifier">Verifier</span>.
 You'll be playing all three roles. In the real world, these actors would be running different services, so we set up different folders to mimic this separation.
 
-These three actors will be exchanging various objects.
-The most important one is the `credential`.
-This is how a `credential` is created:
+Both the <span class="label-role verifier">Verifier</span> and the <span class="label-role attester">Attester</span> have to interact with the KILT blockchain.
+But only the <span class="label-role attester">Attester</span> is required to own KILTs since he has to pay for storing attestation on chain.
```

Comment:
```suggestion
But only the <span class="label-role attester">Attester</span> is required to own KILTs since they have to pay for storing attestation on chain.
```